src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -10,10 +10,10 @@
 
 :where(:root, .#{$menu-toggle}) {
   --#{$menu-toggle}--ColumnGap: var(--pf-t--global--spacer--sm);
-  --#{$menu-toggle}--PaddingBlockStart: var(--pf-t--global--spacer--sm);
-  --#{$menu-toggle}--PaddingInlineEnd: var(--pf-t--global--spacer--sm);
-  --#{$menu-toggle}--PaddingBlockEnd: var(--pf-t--global--spacer--sm);
-  --#{$menu-toggle}--PaddingInlineStart: var(--pf-t--global--spacer--sm);
+  --#{$menu-toggle}--PaddingBlockStart: var(--pf-t--global--spacer--control--vertical--default);
+  --#{$menu-toggle}--PaddingInlineEnd: calc(var(--pf-t--global--spacer--control--horizontal--default) / 2);
```

Comment:
looks like the inline padding for menu toggles is getting a calc (/2) where it shouldn't. should just have the default token value so that the inline padding matches the form controls!